src/cryptominisat.h.in

```diff
@@ -92,8 +92,22 @@ namespace CMSat {
 
         void set_num_threads(unsigned n); //Number of threads to use. Must be set before any vars/clauses are added
         void set_allow_otf_gauss(); //allow on-the-fly gaussian elimination
-        void set_max_time(double max_time); //max time to run to on next solve() call
-        void set_max_confl(int64_t max_confl); //max conflict to run to on next solve() call
+        /**
+         * CPU time (in seconds) that can be consumed before the next call to solve() must return
+         *
+         * Because the elapsed CPU time depends on both the number of
+         * threads, and the activity of these threads, the elapsed time
+         * can wildly differ from wall clock time.
+         * 
+         * Note: Negative values are ignored.
+         */
+        void set_max_time(double max_time);
+        /**
+         * Conflicts that can be consumed before the next call to solve() must return
+         *
+         * Note: Negative values are ignored.
```

Comment:
Again, please remove this, no need. Assert-fail is the way. Hard fail is a lot better than soft fail.